### PR TITLE
[Learn] Learning path page sidebar component

### LINKED ISSUE #1984: [Learn] Add sidebar component to learn path page
**Current State:**
The sidebar component has not been built

Screenshot 2021-07-09 at 9 44 52 AM

**Desired State:**

Build the sidebar component taking inspiration from [storybook site](https://storybook.js.org/tutorials/intro-to-storybook/react/en/using-addons/)

**Note**

Use the color scheme of Layer5

---
**Contributor Resources**
The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- Check the Learn Layer5 site [Design Spec](https://docs.google.com/document/d/1tQ9tLbetBVKP7z76FR19vsxbAfoFHZzZYxCt7lJpAhU/edit) to know more about the site structure and development plans.
- Checkout the Learn Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/?node-id=4853%3A0).

### PR DESCRIPTION
**Pull request by sudo-NithishKarthik**
feat: Add sidebar component for learning path page

**Description**
Implements the sidebar component for learning path page

This PR fixes #1984 

**Notes for Reviewers**
I had to make [this change](https://github.com/layer5io/layer5/pull/2008/files#diff-f46efae737b3eb748d1007a9f695eabb7270fb423cf5115590c090d09cd056efR24) since there was not enough space and was looking very cluttered.

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by sudo-NithishKarthik** [comment]
https://user-images.githubusercontent.com/75248557/126200071-e03f278b-d058-4c31-8824-0f49e3acf87d.mov

**Comment by l5io** [comment]
🚀 Preview for commit 37b408d851db0cff8361cbd222891d061be55989 at: https://60f5b665f4d13b992c77f2c0--layer5ng.netlify.app

**Comment by leecalcote** [comment]
ooooh, nice.

**Comment by l5io** [comment]
🚀 Preview for commit 56b7ecf6a0ac649bfda89ac3c73fcdb86d62a7ab at: https://60f5b96ac0d5dd5f87aa677e--layer5ng.netlify.app

File: src/components/Learn-Components/TOC-Learning-Path/index.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+          <HiOutlineChevronLeft />
+          <h4>Learn-Layer5</h4>
+        </Link>
+      </div>*/}
+      <div className="toc-list">
+        <ul>
+          <h5 className="toc-sub-heading">
+            Pre-requisites
+          </h5>
+          {[{name: "Set up", link: "/service-mesh-management/meshery"},{name:"Run Meshery",link:"/service-mesh-management/meshery/getting-started"},{name: "Manage Meshery", link: "/service-mesh-management/meshery/operating-service-meshes"}].map(item =>
```
Comment (Nikhil-Ladha) [inline src/components/Learn-Components/TOC-Learning-Path/index.js:28]: These links should be a href to the sections within the page, and not point to the actual page.
Also, better to have these in a variable above and then use that variable here.

File: src/components/Learn-Components/TOC-Learning-Path/index.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+          )}
+
+          <h5 className="toc-sub-heading">
+            Courses
+          </h5>
+
+          {coursesData.map((item) => (
+            <li key={item}>
+              <p className="toc-item">
+                <a href={`/${item.fields.slug}`}>
```
Comment (Nikhil-Ladha) [inline src/components/Learn-Components/TOC-Learning-Path/index.js:45]: Same as above

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
Also, the other change wrt to design I have communicated to you on slack, do check that.

**Comment by Nikhil-Ladha** [comment]
Hey @sudo-NithishKarthik any updates here?

**Comment by sudo-NithishKarthik** [comment]
Almost done, will push it soon 👍🏽

**Comment by sudo-NithishKarthik** [comment]
Screenshot 2021-07-25 at 12 55 06 PM

**Comment by l5io** [comment]
🚀 Preview for commit 12faf29fa5f8a43492afb71d3b9eb3ff7e938b41 at: https://60fd13d7e0e9a44a2582f1a7--layer5ng.netlify.app

**Comment by Nikhil-Ladha** [comment]
Looks perfect!
Just 1 small change, a bit unrelated though. Can you also change the `Go Back`Button to a `< Learning Paths` Button, as we have on this [page](https://60fd13d7e0e9a44a2582f1a7--layer5ng.netlify.app/learn-ng/mastering-meshery-istio/introduction-to-service-meshes/istio/getting-started). Also, make the sidebar and the right section aligned at the same level.

**Comment by sudo-NithishKarthik** [comment]
Yes @Nikhil-Ladha , sure, i was thinking of that as well 😄

**Review by leecalcote — APPROVED** [review]

**Comment by sudo-NithishKarthik** [comment]
Screenshot 2021-07-26 at 10 45 41 AM

**Comment by l5io** [comment]
🚀 Preview for commit ae8ef4a11cb3b48892af0dc0cea011428ed07cb6 at: https://60fe46ba571f06af9f99622e--layer5ng.netlify.app

**Comment by Nikhil-Ladha** [comment]
Sorry for not mentioning it earlier, but 2 more small change is required that being:
The Headings of the section should also be hyperlinked i.e, `Couses` and `Pre-requisites`, and also change `Get Started` to `Pre-requisites`.
And, other thing is `Go Back` can be renamed as `Learning Paths`.

**Comment by sudo-NithishKarthik** [comment]
Screenshot 2021-07-26 at 5 12 49 PM

**Comment by l5io** [comment]
🚀 Preview for commit 4f016c380a505d406acf781ba4a235ce93867ee3 at: https://60fea1c84106972044fb5520--layer5ng.netlify.app

**Review by Nikhil-Ladha — APPROVED** [review]

**Pull request merged**